gobblin-core/src/test/java/gobblin/source/extractor/hadoop/AvroFsHelperTest.java

```diff
@@ -27,7 +27,7 @@
 
 public class AvroFsHelperTest {
 
-  @Test(expectedExceptions = IllegalArgumentException.class)
```

Comment:
What exception does it throw? Is it possible to add this to expectedExceptions instead of disabling the test?